src/status_im/contexts/shell/activity_center/notification/news/view.cljs

```diff
@@ -0,0 +1,82 @@
+(ns status-im.contexts.shell.activity-center.notification.news.view
+  (:require [clojure.string :as string]
+            [promesa.core :as promesa]
+            [quo.core :as quo]
+            [react-native.core :as rn]
+            [status-im.contexts.shell.activity-center.notification.common.view :as common]
+            [utils.datetime :as datetime]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn auto-resized-image
+  [url]
+  (let [[height set-height] (rn/use-state nil)
+        window-width        (- (:width (rn/get-window)) 40)]
+    (rn/use-effect #(-> (rn/image-get-size url)
+                        (promesa/then (fn [[w h]]
+                                        (let [scale      (/ window-width w)
+                                              new-height (* h scale)]
+                                          (set-height new-height))))))
+    (if height
+      [rn/image
+       {:resize-mode :contain
+        :style       {:width         window-width
+                      :height        height
+                      :align-self    :center
+                      :border-radius 12}
+        :source      url}]
+      [rn/view {:style {:height 200 :align-items :center :justify-content :center}}
+       [rn/activity-indicator]])))
+
+(defn sheet
+  [{:keys [newsImageUrl newsTitle newsContent newsLink newsLinkLabel]} timestamp]
```

Comment:
Notification keys can be renamed in the existing `data_store/activities.cljs`. file. Could you rename these news feed keys as well there?

https://github.com/status-im/status-mobile/blob/e50c49b43707860d751da4136595e565fbb0d6fd/src/legacy/status_im/data_store/activities.cljs#L60-L67